apps/studio/components/interfaces/Auth/EmailTemplates/SpamValidation.tsx

```diff
@@ -28,33 +38,38 @@ export const SpamValidation = ({ validationResult }: SpamValidationProps) => {
         hasSpamWarning ? (
           <>
```

Comment:
this fragment is probably unnecessarily now

Reply:
spamRules.length > 0 has the same check as hasSpamWarning too i realised - will push a refactor